brands/amenity/cafe.json

```
@@ -938,6 +949,16 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|Secret Recipe": {
+     "tags": {
+       "amenity": "cafe",
+       "brand": "Secret Recipe",
+       "brand:wikidata": "Q7444079",
+       "brand:wikipedia": "en:Secret_Recipe_(restaurant)",
```

Make sure to convert underscores to spaces in the `brand:wikipedia` tag. In this case, as it's a Malaysian brand, it might even be more appropriate to link to the Malaysian Wikipedia page.
```suggestion
       "brand:wikipedia": "en:Secret Recipe (restaurant)",
```

---

Thanks for reminding. I would prefer to English Wikipedia since this brand can be found worldwide, not only in Malaysia.